query/src/sql/planner/binder/project.rs

```diff
@@ -74,9 +75,17 @@ impl<'a> Binder {
     pub(super) async fn normalize_select_list(
         &mut self,
         select_list: &[SelectTarget<'a>],
-        input_context: &BindContext,
+        table_alias: &Option<TableAlias<'a>>,
```

Comment:
What happen if this query:
```
SELECT * FROM system.columns, (SELECT 1) AS `system.columns`;
```

Reply:
Duplicated table names will result in an error.

Reply:
\`system.table\` should not be equal to system.table

https://www.db-fiddle.com/f/k2vZTW2y8Vm9AjUSTPkVGv/1

Reply:
> `system.table` should not be equal to system.table
> 
> https://www.db-fiddle.com/f/k2vZTW2y8Vm9AjUSTPkVGv/1

The "system.table" should be parsed into identifier. It won't be an issue.